test/html.tsx

```diff
@@ -241,6 +241,42 @@ describe('HTML parser for mrkdwn', () => {
       expect(html(<code>｀code｀</code>)).toBe('`\u02cbcode\u02cb`')
     })
 
+    it('allows containing link', () => {
+      expect(
+        html(
+          <code>
+            <a href="https://example.com/">example</a>
+          </code>
+        )
+      ).toBe('`<https://example.com/|example>`')
+
+      // Raw mrkdwn via JSX interpolation
+      expect(html(<code>{'<#C01234567>'}</code>)).toBe('`<#C01234567>`')
+
+      // Work escape correctly
+      expect(html(<code>&lt;!channel&gt;</code>)).toBe('`&lt;!channel&gt;`')
```

Comment:
It seems to return same result as `<code>{'<!channel>'}</code>`. jsx-slack could not detect whether using escaped entity like `&lt;` and `&gt;`. This test case is still valid in using entities via JSX interpolation, like `<code>{'&lt;!channel&gt;'}</code>`.